src/runtime/internal/style_manager.ts

```diff
@@ -18,6 +18,12 @@ function hash(str: string) {
 	return hash >>> 0;
 }
 
+const create_style_information = (doc: Document | ShadowRoot, node: Element & ElementCSSInlineStyle) => {
```

Comment:
I propose to make this a function to adhere to the rest of the code style in this file